packages/core/src/domain/configuration/transportConfiguration.ts

```diff
@@ -54,18 +62,23 @@ export function computeTransportConfiguration(
       useAlternateIntakeDomains: true,
       intakeApiVersion: 2,
     }
-    const replicaEndpointBuilder = createEndpointBuilder(replicaConfiguration, buildEnv)
-
-    configuration.replica = {
-      applicationId: initConfiguration.replica.applicationId,
-      internalMonitoringEndpoint: replicaEndpointBuilder.build('logs', 'browser-agent-internal-monitoring'),
-      logsEndpoint: replicaEndpointBuilder.build('logs'),
-      rumEndpoint: replicaEndpointBuilder.build('rum'),
+    const replicaEndpointBuilders = {
+      logsEndpointBuilder: createEndpointBuilder(replicaConfiguration, buildEnv, 'logs'),
+      rumEndpointBuilder: createEndpointBuilder(replicaConfiguration, buildEnv, 'rum'),
+      internalMonitoringEndpointBuilder: createEndpointBuilder(
+        replicaConfiguration,
+        buildEnv,
+        'logs',
+        'browser-agent-internal-monitoring'
+      ),
     }
+    configuration.replica = { applicationId: initConfiguration.replica.applicationId, ...replicaEndpointBuilders }
 
-    const replicaIntakeUrls = ENDPOINTS_TYPES.map((endpointType) => replicaEndpointBuilder.buildIntakeUrl(endpointType))
-    replicaIntakeUrls.forEach((replicaIntakeUrl) => intakeUrls.push(replicaIntakeUrl))
-    intakeUrls.push(...replicaIntakeUrls)
+    const replicaIntakeEndpoints: string[] = objectValues(replicaEndpointBuilders).map((builder) =>
+      builder.buildIntakeUrl()
+    )
+    replicaIntakeEndpoints.forEach((replicaIntakeEndpoint) => intakeEndpoints.push(replicaIntakeEndpoint))
+    intakeEndpoints.push(...replicaIntakeEndpoints)
```

Comment:
not related to your change but it seems that we are adding the replica endpoints twice

Reply:
good catch